games/godot/Portfile

```diff
@@ -22,10 +22,38 @@ long_description    Godot Engine is a cross-platform game engine for \
                     (Linux, macOS, Windows), as well as mobile \
                     (Android, iOS) and web-based (HTML5) platforms.
 
-checksums           rmd160  4ff4f9c4798d67cf6cf4d63c38bed84d400daa8e \
+if {$subport eq ${name}} {
+    github.setup    godotengine ${name} 3.3 "" -stable
+    version         3.3.0
```

Comment:
why change the version number, from the upstream tag?

Reply:
> why change the version number, from the upstream tag?

Mostly just to stick to semantic versioning. Godot seems to drop the patch number whenever they reach a zero. It's basically the same as asking why MacPorts adds revision 0 to the version numbers by default, e.g. 3.2.3_0. Why not only add a revision when it gets set to 1 or higher?

> Also why do you put this in a if block? Can this not just be part in the "main" part of the Portfile as it was before and then you just add a subport for the earlier version?

If it works without needing to be inside of an `if` block, I'm certainly not against it. I'm not really an expert on the nuances of how subports work, so I simply copied what I saw from other portfiles.